java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/user/MapApiController.java

```diff
@@ -501,4 +519,31 @@ public ResponseEntity<Resource> downloadObf(HttpServletResponse response, @Reque
 			}
 		}
 	}
+	
+	@GetMapping(path = { "/get-account-info" })
+	@ResponseBody
+	public ResponseEntity<String> getAccountInfo() {
+		final String ACCOUNT_KEY = "account";
+		final String FREE_ACCOUNT = "Free";
+		final String PRO_ACCOUNT = "Osmand_Pro";
```

Comment:
We have different subspcritions "OsmAnd PRO" (check name) amazon, Huawei, Apple, Google, Promo